lib/copy-sync/__tests__/copy-sync-dir.test.js

```diff
@@ -121,29 +147,61 @@ describe('+ copySync()', () => {
 
       assert(!fs.existsSync(path.join(dest, IGNORE)), 'directory was not ignored')
       assert(!fs.existsSync(path.join(dest, IGNORE, 'file')), 'file was not ignored')
+      done()
```

Comment:
Revert this change. Using `done` isn't needed since we are using sync methods here.